lib/vrl/stdlib/src/parse_groks.rs

```diff
@@ -141,6 +146,33 @@ impl Function for ParseGroks {
             })
             .collect::<std::result::Result<BTreeMap<String, String>, vrl::function::Error>>()?;
 
+        let alias_sources = arguments
+            .optional_array("alias_sources")?
+            .unwrap_or_default()
+            .into_iter()
+            .map(|expr| {
+                let path = expr
+                    .as_value()
+                    .ok_or(vrl::function::Error::ExpectedStaticExpression {
+                        keyword: "alias_sources",
+                        expr,
+                    })?
+                    .try_bytes_utf8_lossy()
+                    .expect("filename not bytes")
+                    .into_owned();
+                Ok(path)
+            })
+            .collect::<std::result::Result<Vec<String>, vrl::function::Error>>()?;
+
+        for src in alias_sources {
+            let path = Path::new(&src);
+            let file = File::open(&path).unwrap();
```

Comment:
This needs to be handled properly and an InvalidAliasSource should be returned, but I'm not sure what the proper way to do this is.